scheduled_scripts/roles.js

```diff
@@ -12,7 +12,6 @@ module.exports = {
     init: function(client, config) {
         this.client = client;
         this.configFile = config;
-        // this.roleMessage();
```

Comment:
I'm keeping this line for now, as this pushes out the initial message. It is commented out so reboots don't push new ones.

Reply:
That's okay